modules/apps/site/site-memberships-web/src/main/java/com/liferay/site/memberships/web/internal/portlet/SiteMembershipsPortlet.java

```diff
@@ -388,6 +388,18 @@ public void unassignUserGroupGroupRole(
 			userGroupId, group.getGroupId(), roleIds);
 	}
 
+	public void unassignUserRole(
```

Comment:
Can you rename `userRole` to `userGroupRole` for clarity? This happens in other places as well (eg `unassignUserRoleURL`, `unassignUserRoleFm`).